cmd/notary/tuf.go

```diff
@@ -246,6 +256,10 @@ func (t *tufCommander) tufInit(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Must specify a GUN")
 	}
 
+	if t.rootCert != "" && t.rootKey == "" {
+		return fmt.Errorf("--rootCert specified without --rootKey being specified")
```

Comment:
@riyazdf Fixed in latest push https://github.com/docker/notary/pull/821/commits/c2baf9e005dfecc03c9d970bd0f11187feaa7e1d